src/presentation/controller/registration/Routing.ts

```diff
@@ -12,12 +12,15 @@ const START_URL = `${BASE_URL}/start`;
 export const WHICH_TYPE_TEMPLATE = RegistrationPageType.whichType;
 export const NAME_TEMPLATE = RegistrationPageType.name;
 export const NEXT_TEMPLATE = RegistrationPageType.next;
+export const GENERAL_PARTNERS_TEMPLATE = RegistrationPageType.generalPartners;
 
 export const WHICH_TYPE_URL = `${BASE_URL}/${WHICH_TYPE_TEMPLATE}`;
 export const NAME_URL = `${BASE_URL}/${NAME_TEMPLATE}`;
 
 export const NEXT_URL = `${BASE_URL}/transaction/${TRANSACTION_ID}/submission/${SUBMISSION_ID}/${NEXT_TEMPLATE}`;
 
+export const GENERAL_PARTNERS_URL = `${BASE_URL}/${GENERAL_PARTNERS_TEMPLATE}`;
```

Comment:
Not sure whether we need the transaction and submission id here.

Reply:
yes we will need both id's in the url. Think of it as any page after the submission gets created in MongDB (name page?) will need the id's in the url so we know where to get and read data from.

Reply:
yes, need ids
and can you move it to line 19 to follow the journey order

Reply:
Added the ids in.